### ISSUE TITLE
Can this work for normal react.js + meteor?

### ISSUE THREAD

**Issue by gitTerebi** [inProgress-team/react-native-meteor#106]
How hard would it be to get this working for normal reactjs?

Or are there any other alternatives.

So far I haven't found anything that works, the github examples are broken.

**Comment by funkyeah** [comment]
Not sure what you are getting at. 

Part of the point of react-native is that parts of react won't fully work on native platforms (at least not without using webviews). If you are not targeting native mobile platforms then you can you use normal react with just plain meteor.

If you are having an issue with an example you should provide a little context to the issue(s) you are having.

**Comment by gitTerebi** [comment]
I mean i'm developing a fully reactjs front end using webpack. There is no meteor there.

Instead of using ddp.js it would be nice to have access to meteor esque methods in my react app.

**Comment by smooJitter** [comment]
That's the purpose of this package. It allows you to access meteor data management functions. Pub/sub and methods, it also proves access to Meteor.users

**Comment by funkyeah** [comment]
But it certainly implies the usage of Meteor at least on the backend. 

I am having a tough time understanding why you would be looking at a package titled `react-native-meteor` if you are not building a react-native application and are not using Meteor...

**Comment by Mokto** [comment]
No this doesn't work for not react-native apps. Use this list : http://meteorpedia.com/read/DDP_Clients

**Comment by bvodola** [comment]
I know this is an old issue, but I just [forked this repo and changed it](https://github.com/bvodola/react-meteor-client) a little bit to make react-native-meteor work with 'normal' React. For every `react-native` dependency, I changed it for `react-native-web`. Still a work in progress, but I managed to complete [this tutorial](https://medium.com/differential/connecting-react-native-and-meteor-cccdb0df29e3), switching from `react-native-meteor` to my fork, which is called `react-meteor-client`. Here's the link for it: [https://github.com/bvodola/react-meteor-client](https://github.com/bvodola/react-meteor-client). Hope it helps.
